com.github.kagami.boram.yml

```diff
@@ -0,0 +1,241 @@
+app-id: com.github.kagami.boram
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: Boram
+finish-args:
+  # X11 + XShm access
+  - --share=ipc
+  - --socket=x11
+  # Wayland access
+  - --socket=wayland
+  # Needs to talk to the network:
+  - --share=network
+  # Needs to save files locally
+  - --filesystem=home
+  - --filesystem=/mnt
+  - --filesystem=/media
+  - --device=dri
+  - --socket=pulseaudio
+modules:
+  - shared-modules/gtk2/gtk2.json
+  - shared-modules/dbus-glib/dbus-glib.json
+
+  - name: boram
+    buildsystem: simple
+    build-commands:
+      - install -D boram.sh /app/bin/Boram
+      - install -d /app/lib/Boram
+      - install -Dm644 -t /app/share/applications/ com.github.kagami.boram.desktop
+      - install -Dm644 resources/app/icon-big.png /app/share/icons/com.github.kagami.boram.png
+      - install -Dm644 -t /app/share/metainfo/ com.github.kagami.boram.metainfo.xml
+      - cp -r * /app/lib/Boram
+    sources:
+      - type: file
+        path: boram.sh
+      - type: archive
+        url: https://github.com/Kagami/boram/releases/download/v0.5.6/boram-v0.5.6-linux-x64.7z
```

Comment:
You are still building an aarch64 but extracct binaries for x86_64 only.

Also 7z....

Reply:
The 7zip format is the only one provided through the official releases page.

Reply:
maybe you should tell them to not use it for Linux

Reply:
The developer hasn't been interacting with the repo recently, so I believe that would take a while to happen.

It builds well extracting from 7zip, though.